src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -349,17 +349,147 @@ <h3 *ngIf="displaySubsection(showQuestion, 4, 7)">Managing Session Responses</h3
+                      </option>
+                      <option [ngValue]="InstructorSessionResultSectionType.GIVER">
+                        Show response if the giver is in the selected section
+                      </option>
+                      <option [ngValue]="InstructorSessionResultSectionType.EVALUEE">
+                        Show response if the evaluee is in the selected section
+                      </option>
+                      <option [ngValue]="InstructorSessionResultSectionType.BOTH">
+                        Show response only if both are in the selected section
+                      </option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+              <div class="col-md-6">
+                <label class="control-label">
+                  Additional settings:
+                </label>
+                <div ngbTooltip="Group results in the current view by team">
+                  <input type="checkbox"> Group by Teams
+                </div>
+                <div ngbTooltip="Show statistics">
+                  <input type="checkbox"> Show Statistics
+                </div>
+                <div ngbTooltip="Indicate missing responses">
+                  <input type="checkbox"> Indicate Missing Responses
+                </div>
+              </div>
+            </div>
+          </tm-example-box>
           <p>
             In the example below, results are sorted by <b>Giver > Recipient > Question</b>. Additionally, missing responses
             have been recorded.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <tm-instructor-session-result-grq-view [session]="this.exampleFeedbackSession"
+                                                   [section]="'Section A'"
+                                                   [sectionType]="InstructorSessionResultSectionType.EITHER"
+                                                   [groupByTeam]="true"
+                                                   [showStatistics]="true"
+                                                   [indicateMissingResponses]="true"
+                                                   [isExpandAll]="true"
+                                                   [instructorCommentTableModel]="exampleInstructorCommentTableModel"
+                                                   [responses]="exampleGrqResponses"
+            ></tm-instructor-session-result-grq-view>
+            <div class="card-header bg-info text-dark course-tab-header">
+              Participants who have not responded to any question
+              <div class="float-right">
+                <i class="fas fa-chevron-up"></i>
+              </div>
+              <div class="float-right" style='padding-right:15px; margin:-5px;'>
+                <button class="btn btn-light btn-sm button" type="button">
+                  Remind All
+                </button>
```

Comment:
Following the previous comment, this button should be `disabled` as well?

Reply:
Must have missed it. Thanks :)

However, I think we should disable the buttons by making them do nothing BUT make them clickable, instead of making them non-clickable. Your thoughts?

Reply:
See comments below. In fact I also encounter similar problem elsewhere :)